packages/apps/docs/src/docs/deploy/chainweb/chainweb-get-started.md

```diff
@@ -22,13 +25,479 @@ Before installing Chainweb, you should verify that your computer meets the follo
+   
+   Note that the archive file naming convention includes the `chainweb-node` version, compiler version, Ubuntu version, and a commit hash identifier using the following format:
+   
+   ```text
+   chainweb-<version>.ghc-<version>.ubuntu-<version>.<revision>.tar.gz
+   ```
+   
+   For example, the `chainweb-node` archive file for Ubuntu 22.04 looks like this:
+   
+   ```text
+   chainweb-2.24.1.ghc-9.6.5.ubuntu-22.04.89b0ac3.tar.gz
+   ```
+
+5. Change to the directory that contains the downloaded file—typically, the Downloads folder—or move the file to a new location.
+
+6. Unzip and extract the compressed archive by running a command similar to the following:
+   
+   ```bash
+   tar -xvzf chainweb-2.24.1.ghc-9.6.5.ubuntu-22.04.89b0ac3.tar.gz
+   ```
+
+7. Verify the binary is ready to use and review command-line configuration options by running the following command:
+   
+   ```bash
+   ./chainweb-node --help
+   ```
+
+   You should see usage information about the configuration settings you can specify as command-line options similar to the following truncated output:
+
+   ```bash
+   Usage: chainweb-node [--info] [--long-info] [-v|--version] [--license] 
+                        [-?|-h|--help] 
+                        [--print-config-as full|minimal|diff | --print-config] 
+                        [--config-file FILE] 
+   ```
+
+   From the usage information, you can see that there are a large number of configuration options that you can use to control and operation and behavior of the Chainweb node. 
+   Before you start the node, you should review the configuration options and the default values to determine whether you want to make any changes to the configuration of the node.
+   For information about this next step, see [Review the default configuration](#review-the-default-configuration).
+
+## Run in a Docker container
+
+If you have Docker installed, you can download a `chainweb-node` image and run a node in a Docker container.
+If you want to run the node in a Docker container but don't have Docker installed yet, see the instructions provided in [Install Docker Engine](https://docs.docker.com/engine/install/) to download and install Docker for your operating system.
+After you have Docker installed, use the following instructions to pull the Docker image and run `chainweb-node` in a Docker container.
+
+To run `chainweb-node` in a Docker container:
+
+1. Open a terminal shell on the physical or virtual host where you have Docker installed.
+
+2. Verify that Docker is installed and the `docker` process is running with the following command:
+   
+   ```bash
+   docker --version
+   ```
+
+   This commands should return information similar to the following:
+
+   ```bash
+   Docker version 24.0.6, build ed223bc
+   ```
+
+3. Pull the latest image by running the following command:
+   
+   ```bash
+   docker pull ghcr.io/kadena-io/chainweb-node/ubuntu:latest
+   ```
+
```

Comment:
This reminds me: it may be worth noting, probably not here, that nodes must periodically be updated to keep up with the latest hard forking changes to the network.